README.md

```diff
@@ -1 +1,26 @@
-Terraform configuration for kafka clusters to create topis and ACLs
+# Terraform configuration for kafka clusters to create topis and ACLs
+
+## how to plan/ apply shared Kafka Terraform resources?
+
+1. Login to AWS- run `make okta-aws-cli-login` for profile `pubsub-admin` from the `terraform` repo dir. More
+   info [here](https://github.com/utilitywarehouse/terraform/tree/master/aws#okta-login)
+
+```
+[pubsub]
+region         = eu-west-1
+role_arn       = arn:aws:iam::950135041896:role/pubsub-admin
+source_profile = uw-prod-jump-role
+```
+
+Profile must be referenced while performing next actions, for example by running `export AWS_PROFILE=pubsub` in each
+terminal window.
+
+2. Go to directory with the terraform state- e.g. `cd dev-aws/kafka-shared`
+3. Run `make init` to initialize terraform state. This needs to happen only once per directory.
+4. In a separate window but in the samer directory run `make connect` in order to connect to the Kafka cluster. <br>
```

Comment:
```suggestion
4. In a separate window but in the same directory run `make connect` in order to connect to the Kafka cluster. <br>
```

Reply:
[Applied](https://github.com/utilitywarehouse/kafka-cluster-config/pull/116/commits/0720bad09b454b92879a665432c44fa2b657a2a9#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R25)- together with your suggestion from diff, it looks much better now, thanks!